### DISCUSSION TITLE
Resolve symlinks when previewing files

### DISCUSSION THREAD

**Discussion by chris-sophos** [community/community#54449] [Repositories]
### Select Topic Area

Product Feedback

### Body

When viewing symbolic links, a preview of the symlink's path is shown, along with the recognition that it's a symlink. I think it'd be nice to attempt to resolve the symlinks as they would be resolved in the filesystem, and possibly also support behavior similar to markdown links (linking to other branches or repositories).

**Comment by corneliusroemer** [comment]
I made a more complete discussion item here, showing screenshots of the current state:
- #175854
